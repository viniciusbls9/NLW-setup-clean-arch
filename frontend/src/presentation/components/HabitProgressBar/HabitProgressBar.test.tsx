import { describe, test, expect } from 'vitest';
import { render } from '@testing-library/react';
import '@testing-library/jest-dom';
import { HabitProgressBar } from '.';

describe('HabitProgressBar Component', () => {
  test('Should render correctly HabitProgressBar component', () => {
    const { container } = render(<HabitProgressBar progress={75} />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
