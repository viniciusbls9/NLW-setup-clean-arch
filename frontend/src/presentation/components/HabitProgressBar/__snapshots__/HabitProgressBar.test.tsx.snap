// Vitest Snapshot v1

exports[`HabitProgressBar Component > Should render correctly HabitProgressBar component 1`] = `
<div
  class="h-3 rounded-xl bg-zinc-700 w-full mt-4"
>
  <div
    aria-label="Progresso de hÃ¡bitos completados nesse dia"
    aria-valuenow="75"
    class="h-3 rounded-xl bg-violet-600 w-3/4"
    role="progressbar"
    style="width: 75%;"
  />
</div>
`;
